docs: MaaS introduction enhancement (#1272)
spectrocloud/librarium: content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

# Overview 

The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. The primary function of the PCG is to facilitate connectivity between Palette and MAAS, even if MAAS is behind a NAT gateway or a firewall. The PCG "dials home" to Palette and establishes a permanent connection, traversing any NAT gateways or firewalls, rather than Palette attempting to connect to MAAS directly and being blocked by a firewall. In this way, the function of the PCG is somewhat similar to that of a reverse proxy.

> 

Reviewer: I would move lines 24 and 26 here as an intro to your process bullet list below:

"Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation, or jump box. You can use the installer on any Linux x86-64 system with a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint. 

You can set up the PCG as a single- or three-node cluster based on your requirements for high availability (HA)."

At a high level, the following occurs during a successful MAAS PCG installation:

> - Run a PCG installer Docker image on a laptop, workstation, or jump box.

Reviewer 2: ```suggestion
- Run the PCG installer on a laptop, workstation, or jump box.
```

- provide information to the installer so that it can connect both to a local MAAS installation and to a Palette account
- the installer will use MAAS to obtain machine(s), and then install a PCG on them

> - the PCG will now facilitate all communication between Palette and MAAS, enabling Palette to create new clusters on machines provided by MAAS

Reviewer: ```suggestion
- The PCG then facilitates all communication between Palette and MAAS, enabling Palette to create new clusters on machines that MAAS provides.
```

2. Use the Docker image to start the installation on the installer host.

3. The installer prompts you for information, including the pairing code you obtained in step **1**.

4. The installer generates the PCG configuration file from information you provide in step **3**.

    <br />

    The installer needs access to your Palette account and to your MAAS environment. Additionally, one (no HA) or three (HA) machines must be in ready state and have internet access in MAAS. If you select one machine in step 3, then you need one in MAAS. Likewise, if you select three machines in step 3, you need three in MAAS.
    <br />

> 5. The installer installs to the MAAS machine(s) and uses the configuration file to build a new cluster to host the PCG application.

Reviewer 3: ```suggestion
5. The installer installs the MAAS machines and uses the configuration file to build a new cluster to host the PCG application.
```

|**Parameter**| **Description**|
|-------------|----------------|
| **API Endpoint** |Enter the MAAS API endpoint (syntax is important). This can be a domain or IP address. Example: ``http://10.11.12.13:5240/MAAS``.|
| **API Key** |Enter an active MAAS API key to use for authentication.|

<br />

1. When the installer prompts you, select the appropriate option for each of the following items to define which machines should be selected on the MAAS server for deployment as a PCG:

    - Domain
    - Availability Zone
    - Resource Pool
    - One node (no HA) or three nodes (HA)

> <WarningBox>
> 
> Ensure the MAAS server has one or more machines in the **Ready** state for the chosen Availability Zone and Resource Pool combination.
> 
> </WarningBox>

Reviewer 3: ```suggestion
  <br />
  
  <WarningBox>

  Ensure the MAAS server has one or more machines in the **Ready** state for the chosen Availability Zone and 
  Resource Pool combination.

  </WarningBox>
```

<InfoBox>

> The ``/opt/spectrocloud`` folder is volume mapped to the ``/tmp`` folder on the installer.

Reviewer 3: ```suggestion
The **/opt/spectrocloud** folder is volume mapped to the installer's **/tmp** folder.
```

4. Change the number of nodes to 3.

> Two new nodes will be added to the PCG cluster.

Reviewer 3: ```suggestion
Two new nodes will be added to the PCG cluster.

<br />

```
